packages/reports/src/Containers/open-positions.tsx

```diff
@@ -295,22 +294,38 @@ export const OpenPositionsTable = ({
                                     <PlaceholderComponent is_loading={is_loading} />
                                 </DataTable>
                             </EmptyPlaceholderWrapper>
-                        </DesktopWrapper>
-                        <MobileWrapper>
-                            <EmptyPlaceholderWrapper component_icon={component_icon} is_empty={is_empty}>
-                                <DataList
-                                    className={className}
-                                    data_source={active_positions}
-                                    footer={totals}
-                                    rowRenderer={mobileRowRenderer}
-                                    getRowAction={getRowAction}
-                                    row_gap={8}
-                                    keyMapper={item => item?.id}
-                                >
-                                    <PlaceholderComponent is_loading={is_loading} />
-                                </DataList>
-                            </EmptyPlaceholderWrapper>
-                        </MobileWrapper>
+                        </div>
+                    )
+                )}
+            </React.Fragment>
+        );
+    }
+
+    return (
+        <React.Fragment>
+            {is_loading ? (
+                <PlaceholderComponent
+                    is_loading={is_loading}
+                    empty_message_component={EmptyTradeHistoryMessage}
+                    component_icon={component_icon}
+                    localized_message={localize('You have no open positions yet.')}
+                />
+            ) : (
+                currency && (
+                    <div className='reports__content'>
+                        <EmptyPlaceholderWrapper component_icon={component_icon} is_empty={is_empty}>
+                            <DataList
```

Comment:
If the only difference between desktop and not-desktop is rendering  <DataTable /> or  <DataLis/>, maybe then it can be smth like this?
        ```
  {is_loading ? (
                <PlaceholderComponent
                    is_loading={is_loading}
                    empty_message_component={EmptyTradeHistoryMessage}
                    component_icon={component_icon}
                    localized_message={localize('You have no open positions yet.')}
                />
            ) : (
                currency && (
                    <div className='reports__content'>
                        <EmptyPlaceholderWrapper component_icon={component_icon} is_empty={is_empty}>
                        {isDesktop ?  <DataTable ......./> : <DataList ..../>}
                        </EmptyPlaceholderWrapper>
                   </div>
                )
            )}
```
Because it seems repeatable